### PR TITLE
[bitnami/redis] Redis slave volume init fix

### PR DESCRIPTION
**Pull request by leozilla**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi @leozilla, thanks for the PR! 

Could we keep as `-c` to be aligned with the master init container?

**Comment by leozilla** [comment]
Hi @dani8art 

Yeah sure I can change it to `-c`. I used `-cx` cos it was also used for the master initContainer command (https://github.com/bitnami/charts/blob/11432868f44e258cd95d06214f92ac0966b4f46f/bitnami/redis/templates/redis-master-statefulset.yaml#L252) but I see that all other `/bin/sh` executions dont use `-x`.

**Comment by leozilla** [comment]
Do u want to have it consistent for all commands? Then I suggest to also remove the `-x` from the master initContainer.

**Comment by dani8art** [comment]
> Do u want to have it consistent for all commands? Then I suggest to also remove the `-x` from the master initContainer.

Yes, could we replace all the instances in those init containers for `-ec`, please?

**Comment by leozilla** [comment]
@dani8art 
Good idea. See https://github.com/bitnami/charts/pull/5059/commits/57454a0a288cd419369dba6665da4b0b552cba62

**Comment by leozilla** [comment]
Seems that the test failure is a false positive, cos I already pumped the version.

**Comment by leozilla** [comment]
Ok I just saw there was a version bump in the meantime in master. I merged master into this branch and again bumped the PATCH version of the chart.

File: bitnami/redis/templates/redis-master-statefulset.yaml [hunk 1]
```diff
@@ -249,7 +249,7 @@ spec:
         imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
         command:
           - /bin/sh
```
Comment (dani8art) [inline bitnami/redis/templates/redis-master-statefulset.yaml:251]: could we use `/bin/bash` for both too?

**Review by dani8art — COMMENTED** [review]
Just a minor comment

**Review by dani8art — APPROVED** [review]
LGTM!

**Pull request merged**